recipe/meta.yaml

```diff
@@ -32,7 +35,7 @@ requirements:
     - python             # [win]
     # required for win32: https://wiki.openssl.org/index.php/Compilation_and_Installation#Windows
     - nasm               # [win32]
-    - perl
+    - perl >=5.20.3.1
```

Comment:
We can just make this `==` and re-render. Otherwise we get all 3 `perl`s. Sorry for the headache with this stuff.